interlok-core/src/main/java/com/adaptris/core/security/XpathBuilder.java

```diff
@@ -134,20 +144,16 @@ public void insert(AdaptrisMessage msg, Map<String, String> pathKeyValuePairs) t
       String xpathValue = entry.getValue();
       node = prepareNode(doc, xpathKey, msg);
       if (node != null) {
-        if (xpathValue.startsWith("<")) {
-          if (isStringNestedNodes(xpathValue)) {
-            try {
-              Node nestedNodes = XmlHelper.stringToNode(xpathValue);
-              Node nestedNodesToImport = doc.importNode(nestedNodes, true);
-              while (node.hasChildNodes()) {
-                node.removeChild(node.getFirstChild());
-              }
-              node.appendChild(nestedNodesToImport);
-            } catch (Exception e) {
-              throw new ServiceException(NON_XML_EXCEPTION_MESSAGE);
+        if (xpathValue.startsWith("<") && isStringNestedNodes(xpathValue)) {
```

Comment:
Just wanted to add the only reason I had these as anotehr nested IF statement was I was thinking that it would only be worth checking messages against the isStringNestedNodes() method only if they start with < as to not run that check on all messages. I didn't stop and think that if the firt condition returns false it wouldn't bother checking the 2nd condition as it's linked with &&.